pyxnat/tests/test_resource_functions.py

```diff
@@ -62,7 +62,21 @@ def test_fdg_quantification():
 def test_bamos_volume():
     r = e1.resource('BAMOS')
     v = r.volume()
-    assert(v == 33599.94964499201)
+    assert(v == 33620.32498628937)
+
+
+def test_bamos_stats():
+    r = e1.resource('BAMOS')
+    v = r.stats()
+    print(sum(v['volume']))
+    assert(sum(v['volume']) == 33620.32429030311)
+    r.bullseye_plot(stats=v)
```

Comment:
Shall we add a dummy check to assert it that worked? E.g.,
```
fig,ax = r.bullseye_plot(stats=v)
assert(isinstance(a,matplotlib.figure.Figure))
```

Reply:
939f706 should fix this.